editor/screen.go

```diff
@@ -80,9 +90,62 @@ func initScreenNew(devicePixelRatio float64) *Screen {
 	widget.ConnectMousePressEvent(screen.mouseEvent)
 	widget.ConnectMouseReleaseEvent(screen.mouseEvent)
 	widget.ConnectMouseMoveEvent(screen.mouseEvent)
+	widget.SetAttribute(core.Qt__WA_KeyCompression, false)
+
 	return screen
 }
 
+
+func (s *Screen) toolTip(text string) {
+	family := editor.font.fontNew.Family()
+	size := editor.font.fontNew.PointSize()
+	ttfont := gui.NewQFont2(family, size, int(gui.QFont__Normal), false)
+	tooltip.SetFont(ttfont)
```

Comment:
Why not just tooltip.SetFont(editor.font.fontNew)?

Reply:
Thank you.
I didnt know that such a simple implementation would be possible. 
I'll fix that way.